### ISSUE TITLE
Figure out code signing for Windows

### ISSUE THREAD

**Issue by grantjbutler** [grantjbutler/obs-layout#25]
I've done a little bit of reading, but code signing on Windows is still a little bit of a mystery to me. I know I need to get a certificate somehow, but, unlike code signing for a macOS app, Microsoft doesn't supply certificates and you apparently need to get them from some third party. I'm not entirely sure what options exist. I've looked at only two or three vendors, but most of them don't list a price, and the one who did said that signing certificates are a couple hundred dollars for a single year.

This isn't preventing users from installing the app on Windows, but it is throwing up a nice warning at the start of the installation process that Windows can't verify the software, so that's fun.

**Comment by NoelDavies** [comment]
Certs are a pain, they're not free. They're an annual cost. It's a pain to do, and my experience wasn't great.

Purely from my experiences, I'd say if you want to sign the app purely for having it play nicely with smart screen, Don't it's not worth it. Unless you have thousands of users etc. 

I reached the point of steadily getting 100+ new users per week(ish) and so it made sense for me. 

If you do want to do it, I'd be happy to show you how and save you the time, hassle and energy that I went through 😂.
Feel free to @ me, and food luck 👀

**Comment by grantjbutler** [comment]
Yeah, that's kind of where I landed. I've already got a developer account with Apple for other projects and such, so it was easy enough to set up code signing there because it's just part of a fee I'm already paying. It could very well be that this is just going to be more effort than it's really worth to try to pull off, so it might not be a bad idea to just defer on this until some later time.
